diff -Nru debianefihelper.old/initramfs-post-update/zz-efihelper debianefihelper/initramfs-post-update/zz-efihelper
--- debianefihelper.old/initramfs-post-update/zz-efihelper	1970-01-01 01:00:00.000000000 +0100
+++ debianefihelper/initramfs-post-update/zz-efihelper	2022-04-04 13:43:13.110444847 +0100
@@ -0,0 +1,2 @@
+#!/bin/sh
+/usr/sbin/efihelper update $1
diff -Nru debianefihelper.old/kernel-postinst/zz-efihelper debianefihelper/kernel-postinst/zz-efihelper
--- debianefihelper.old/kernel-postinst/zz-efihelper	1970-01-01 01:00:00.000000000 +0100
+++ debianefihelper/kernel-postinst/zz-efihelper	2022-04-04 13:43:44.620592759 +0100
@@ -0,0 +1,2 @@
+#!/bin/sh
+/usr/sbin/efihelper install $1
diff -Nru debianefihelper.old/kernel-postrm/zz-efihelper debianefihelper/kernel-postrm/zz-efihelper
--- debianefihelper.old/kernel-postrm/zz-efihelper	1970-01-01 01:00:00.000000000 +0100
+++ debianefihelper/kernel-postrm/zz-efihelper	2022-04-04 13:44:27.135486106 +0100
@@ -0,0 +1,2 @@
+#!/bin/sh
+/usr/sbin/efihelper remove $1
